src/Symfony/Component/ClassLoader/DebugUniversalClassLoader.php

```diff
@@ -16,10 +16,10 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class DebugUniversalClassLoader extends UniversalClassLoader
+class DebugUniversalClassLoader extends ClassLoader
```

Comment:
The DebugUniversalClassLoader should also be removed entirely in favor of the DebugClassLoader (special flavors of the UniversalClassLoader are actually meant to be deprecated at the same time than the UniversalClassLoader itself)